utils/utils.go

```diff
@@ -86,8 +86,8 @@ func CreateStem(pkg string, ver string) string {
 }
 
 func ParseStem(pkgVerStem string) (string, string) {
-	pkg, ver, _ := strings.Cut(pkgVerStem, "-")
-	return pkg, ver
+	parts := strings.Split(pkgVerStem, "-")
+	return strings.Join(parts[:len(parts)-1], "-"), parts[len(parts)-1]
```

Comment:
Is this to allow packages with dashes in their names?

Reply:
Yep! I first noticed with a package in my own personal pkgs, but it would also likely happen with oh-my-posh for example.